sachungo/movies
# Filter movies by release year

### Which issue does this PR relate to?

https://www.pivotaltracker.com/story/show/159733072

### What are the changes implemented in the PR?



- Added the year filter
- Added a check to only show the paginator when the total results of movies to display is greater than the allowed page size
- Changed the filter logic to use `logical AND` instead of `logical OR`
- Handle empty state when the API has no movies to show for the filter criteria

### How should the implementation be QA'd?



- Click the deployment button in the comments of this PR and verify that:
    - You can only select one year for the year filter
    - The paginator is not shown when the total results are less than 10
    - Clearing one type of filter updates the list by movies that meet the remaining filter criteria

### Anything else to note


N/A

### Screenshots and/or gifs


N/A
